services/dy-static-file-server/docker/custom/entrypoint.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+set -o errexit
+set -o nounset
+
+IFS=$(printf '\n\t')
+# This entrypoint script:
+#
+# - Executes *inside* of the container upon start as --user [default root]
+# - Notice that the container *starts* as --user [default root] but
+#   *runs* as non-root user [$SC_USER_NAME]
+#
+echo Entrypoint for stage "${SC_BUILD_TARGET}" ...
+echo   User    : "$(id "$(whoami)")"
+echo   Workdir : "$(pwd)"
+
+
+# expect input/output folders to be mounted
+stat "${INPUT_FOLDER}" > /dev/null 2>&1 || \
+        (echo "ERROR: You must mount '${INPUT_FOLDER}' to deduce user and group ids" && exit 1)
+stat "${OUTPUT_FOLDER}" > /dev/null 2>&1 || \
+    (echo "ERROR: You must mount '${OUTPUT_FOLDER}' to deduce user and group ids" && exit 1)
+
+# NOTE: expects docker run ... -v /path/to/input/folder:${INPUT_FOLDER}
+# check input/output folders are owned by the same user
+if [ "$(stat -c %u "${INPUT_FOLDER}")" -ne "$(stat -c %u "${OUTPUT_FOLDER}")" ]
+then
+    echo "ERROR: '${INPUT_FOLDER}' and '${OUTPUT_FOLDER}' have different user id's. not allowed" && exit 1
+fi
+# check input/outputfolders are owned by the same group
+if [ "$(stat -c %g "${INPUT_FOLDER}")" -ne "$(stat -c %g "${OUTPUT_FOLDER}")" ]
+then
+    echo "ERROR: '${INPUT_FOLDER}' and '${OUTPUT_FOLDER}' have different group id's. not allowed" && exit 1
+fi
+
+echo "listing inputs folder"
```

Comment:
MINOR: not important in here but just FYI, in osparc-simcore images we use always prefix for the echo's so they can be easily parse by graylog
SEE https://github.com/ITISFoundation/osparc-simcore/blob/master/services/web/server/docker/entrypoint.sh#L7